### PR TITLE
Updates unamanged storage yamls for rados ceph and hcp

### PR DESCRIPTION
**Pull request by stevsmit**
https://issues.redhat.com/browse/PROJQUAY-4755

CP to 3.7.

### CODE REVIEW

**Comment by dongboyan77** [comment]
There are not specific storage drivers for ceph and hcp in quay. All s3 compatible storage should use `RadosGWStorage`.
@HammerMeetNail Please correct me if I was wrong

File: modules/operator-unmanaged-storage.adoc [hunk 1]
```diff
@@ -57,23 +57,58 @@ DISTRIBUTED_STORAGE_PREFERENCE:
...
+== Ceph Storage
 [source,yaml]
 ----
 DISTRIBUTED_STORAGE_CONFIG:
-  radosGWStorage:
-    - RadosGWStorage
-    - access_key: access_key_here
+  cephStorage: #storage config name
+    - CephStorage #actual driver
+    - access_key: access_key_here #parameters
```
[the remark this answers is not in the document]
Comment (ibazulic) [reply modules/operator-unmanaged-storage.adoc:84]: This section should be removed.

File: modules/operator-unmanaged-storage.adoc [hunk 2]
```diff
@@ -57,23 +57,58 @@ DISTRIBUTED_STORAGE_PREFERENCE:
...
+      secret_key: secret_key_here
+      bucket_name: bucket_name_here
+      hostname: hostname_here
+      is_secure: 'true'
+      port: '443'
+      storage_path: /datastorage/registry
+DISTRIBUTED_STORAGE_DEFAULT_LOCATIONS: []
+DISTRIBUTED_STORAGE_PREFERENCE: #must contain name of the storage config
+    - hcpStorage
```
Comment (ibazulic) [inline modules/operator-unmanaged-storage.adoc:111]: Same comment as before. We don't have a specific Hitachi driver. All drivers that are S3 compatible can be found here:

https://github.com/quay/quay/blob/ff66a93eb7c1b466b2dffd5e62187d7824e6ccad/storage/__init__.py#L21

These are storage classes that we have in Quay and we support.

**Pull request merged**